microshift_networking/microshift-network-policy/microshift-editing-network-policy.adoc

```diff
@@ -0,0 +1,15 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="microshift-editing-network-policy"]
+= Editing a network policy
+include::_attributes/attributes-microshift.adoc[]
+include::_attributes/common-attributes.adoc[]
+:context: microshift-editing-network-policy
+
+toc::[]
+
+You can edit an existing network policy for a namespace. Typical edits might include changes to the pods to which the policy applies, allowed ingress traffic, and the destination ports on which to accept traffic. The `apiVersion`, `kind`, and `name` fields must not be changed when editing `NetworkPolicy` objects, as these define the resource itself.
+
+//OCP modules, edit using conditionals
+include::modules/nw-networkpolicy-edit.adoc[leveloffset=+1]
```

Comment:
@Srivaralakshmi This is a content port from OCP to the MicroShift docs (see on line 4 of the module info below). Changing filenames and IDs on content ports is out of scope (and not a good idea anyway). I am going to suggest adding some best practices with content ports to the Peer Reviewer checklist, because I see we don't have info on content ports there.